eng/targets/CSharp.ReferenceAssembly.props

```diff
@@ -2,6 +2,6 @@
   <PropertyGroup>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <EnableDefaultCompileItems>false</EnableDefaultCompileItems>
-    <NoWarn>$(NoWarn);CS0618;PUB0001</NoWarn>
+    <NoWarn>$(NoWarn);CS0618;CS0169;PUB0001</NoWarn>
```

Comment:
Where did this start showing up?

Reply:
Agreed, we shouldn't add this.

Reply:
Without this we get:

> Microsoft.AspNetCore.Server.Kestrel.Transport.Libuv.Manual.cs(433,44): error CS0169: The field 'LibuvFunctions.uv_buf_t._field1' is never used [E:\code\aspnet\aspnetcore\src\Servers\Kestrel\Transport.Libuv\ref\Microsoft.AspNetCore.Server.Kestrel.Transport.Libuv.csproj]
Microsoft.AspNetCore.Server.Kestrel.Transport.Libuv.Manual.cs(432,44): error CS0169: The field 'LibuvFunctions.uv_buf_t._field0' is never used [E:\code\aspnet\aspnetcore\src\Servers\Kestrel\Transport.Libuv\ref\Microsoft.AspNetCore.Server.Kestrel.Transport.Libuv.csproj]
Microsoft.AspNetCore.Server.Kestrel.Core.Manual.cs(1488,21): error CS0169: The field 'IntegerDecoder._m' is never used [E:\code\aspnet\aspnetcore\src\Servers\Kestrel\Core\ref\Microsoft.AspNetCore.Server.Kestrel.Core.csproj]
Microsoft.AspNetCore.Server.Kestrel.Core.Manual.cs(1487,21): error CS0169: The field 'IntegerDecoder._i' is never used [E:\code\aspnet\aspnetcore\src\Servers\Kestrel\Core\ref\Microsoft.AspNetCore.Server.Kestrel.Core.csproj]

It isn't clear to me how to best resolve this. It refers to:

https://github.com/aspnet/AspNetCore/blob/48e1ec445f21011086595ad080fb21174265be9d/src/Servers/Kestrel/Transport.Libuv/ref/Microsoft.AspNetCore.Server.Kestrel.Transport.Libuv.Manual.cs#L430-L435

Reply:
Here's the struct in `src`:

https://github.com/aspnet/AspNetCore/blob/30be2bdf76d32683ad3a29d4834cb11199faecb0/src/Servers/Kestrel/Transport.Libuv/src/Internal/Networking/LibuvFunctions.cs#L429-L455